wger/gym/templates/gym/member_list.html

```diff
@@ -56,6 +58,27 @@ <h4>{% trans "Administrators and trainers" %}</h4>
         {{current_user.obj.get_full_name}}
     </td>
 
+    {% comment %} show if trainer is active or inactive {% endcomment %}
+    <td>{% if current_user.obj.is_active %}
+        <span>Yes</span>
+        {% else %}
+        <span>No</span>
+        {% endif %}
+    </td>
+
+    {% comment %} Add activate and deactivate button {% endcomment %}
+    {% if perms.gym.manage_gym or perms.gym.manage_gyms %}
+        <td>
+            {% if not current_user.perms.manage_gym and not current_user.perms.manage_gyms %}
```

Comment:
Nice work, I would suggest though that you add simple comment explaining the logic here.